packages/Checkbox/stories/examples/CheckboxGrouping.js

```diff
@@ -16,13 +21,7 @@ const CheckboxExample = props => {
         Checkbox Grouping
       </Heading>
       <Rule />
-      <div
-        css={`
-          [data-pka-anchor="checkbox"] {
-            margin-bottom: 12px;
-          }
-        `}
-      >
+      <div>
```

Comment:
iei this could be a styled component?

Reply:
I'll demonstrate a few different styling techniques, for inspiration. I don't think best practices have emerged, and for Storybook stories, I don't know that we need to enforce best practices.